src/harness/sas_testcase.py

```diff
@@ -228,13 +226,8 @@ def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
     - If the status is not changed within 2 hours it will throw an exception.
     """
     self._sas_admin.TriggerDailyActivitiesImmediately()
-    signal.signal(signal.SIGALRM,
-                  lambda signum, frame:
-                  (_ for _ in ()).throw(
-                      Exception('Daily Activity Check Timeout')))
-
     # Timeout after 2 hours if it's not completed
```

Comment:
Hm, I think we meant to change this to 24 hours (the meeting minutes from April 24 are not very clear but I think that was the general agreement). Would you mind making that modification now?

Reply:
ok